File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/RefSetsPorIdDescripcion.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for refSetsPorIdDescripcion complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="refSetsPorIdDescripcion">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element ref="{http://service.ws.semantikos.minsal.cl/}peticionRefSetsPorIdDescripcion"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "refSetsPorIdDescripcion", propOrder = {
    "peticionRefSetsPorIdDescripcion"
})
public class RefSetsPorIdDescripcion {

    @XmlElement(namespace = "http://service.ws.semantikos.minsal.cl/", required = true)
    protected PeticionRefSetsPorIdDescripcion peticionRefSetsPorIdDescripcion;

    /**
     * Gets the value of the peticionRefSetsPorIdDescripcion property.
     * 
     * @return
     *     possible object is
     *     {@link PeticionRefSetsPorIdDescripcion }
     *     
     */
    public PeticionRefSetsPorIdDescripcion getPeticionRefSetsPorIdDescripcion() {
        return peticionRefSetsPorIdDescripcion;
    }

    /**
     * Sets the value of the peticionRefSetsPorIdDescripcion property.
     * 
     * @param value
     *     allowed object is
     *     {@link PeticionRefSetsPorIdDescripcion }
     *     
     */
    public void setPeticionRefSetsPorIdDescripcion(PeticionRefSetsPorIdDescripcion value) {
        this.peticionRefSetsPorIdDescripcion = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/refSetsPorIdDescripcion.php
<?php

namespace Semantikos\ClientBundle\API;

class refSetsPorIdDescripcion
{

    /**
     * @var PeticionRefSetsPorIdDescripcion $peticionRefSetsPorIdDescripcion
     */
    protected $peticionRefSetsPorIdDescripcion = null;

    /**
     * @param PeticionRefSetsPorIdDescripcion $peticionRefSetsPorIdDescripcion
     */
    public function __construct($peticionRefSetsPorIdDescripcion)
    {
      $this->peticionRefSetsPorIdDescripcion = $peticionRefSetsPorIdDescripcion;
    }

    /**
     * @return PeticionRefSetsPorIdDescripcion
     */
    public function getPeticionRefSetsPorIdDescripcion()
    {
      return $this->peticionRefSetsPorIdDescripcion;
    }

    /**
     * @param PeticionRefSetsPorIdDescripcion $peticionRefSetsPorIdDescripcion
     * @return refSetsPorIdDescripcion
     */
    public function setPeticionRefSetsPorIdDescripcion($peticionRefSetsPorIdDescripcion)
    {
      $this->peticionRefSetsPorIdDescripcion = $peticionRefSetsPorIdDescripcion;
      return $this;
    }

}
